extensions/amp-analytics/0.1/test/test-linker.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createLinker} from '../linker';
+import {mockWindowInterface} from '../../../../testing/test-helper';
+
+describe('Linker', () => {
+  let sandbox;
+
+  beforeEach(() => {
+    // Linker uses a timestamp value to generate checksum.
+    sandbox = sinon.sandbox;
+    sandbox.useFakeTimers();
```

Comment:
I'm wondering if the timezone can be a source of test flakiness. should you mock out the Date.prototype.getTimezoneOffset.

Reply:
I was hoping that `useFakeTimers` would take care of this, but you are right it does not. Done.